src/main/java/com/gluonhq/substrate/target/AndroidTargetConfiguration.java

```diff
@@ -128,6 +129,34 @@ public boolean link() throws IOException, InterruptedException {
             return false;
         }
 
+        // Prepare files
```

Comment:
This block can be moved to a separate method.

Reply:
Done